lib/models/people.dart

```diff
@@ -0,0 +1,70 @@
+class Person {
+    String birth_year;
+    String created;
+    String edited;
+    String eye_color;
+    List<String> films;
+    String gender;
+    String hair_color;
+    String height;
+    String homeworld;
+    String mass;
+    String name;
+    String skin_color;
+    List<String> species;
+    List<String> starships;
+    String url;
+    List<String> vehicles;
+
+    Person({this.birth_year, this.created, this.edited, this.eye_color, this.films, this.gender, this.hair_color, this.height, this.homeworld, this.mass, this.name, this.skin_color, this.species, this.starships, this.url, this.vehicles});
+
+    factory Person.fromJson(Map<String, dynamic> json) {
+        return Person(
+            birth_year: json['birth_year'], 
+            created: json['created'], 
+            edited: json['edited'], 
+            eye_color: json['eye_color'], 
+            films: json['films'] != null ? new List<String>.from(json['films']) : null, 
+            gender: json['gender'], 
+            hair_color: json['hair_color'], 
+            height: json['height'], 
+            homeworld: json['homeworld'], 
+            mass: json['mass'], 
+            name: json['name'], 
+            skin_color: json['skin_color'], 
+            species: json['species'] != null ? new List<String>.from(json['species']) : null, 
```

Comment:
better use Null aware instead of null checks
https://medium.com/@thinkdigitalsoftware/null-aware-operators-in-dart-53ffb8ae80bb